test/unit/resources/aws_s3_bucket_test.rb

```diff
@@ -29,14 +29,14 @@ def test_constructor_reject_unknown_resource_params
   end
 end
 
-#=============================================================================#
```

Comment:
This re-organization isn't quite how we like to do it.  Usually we have a test class for each major section - so Search/ Recall would be a class, and Properties would be a class as well.